### DISCUSSION TITLE
FR: Search for Untagged Links

### DISCUSSION THREAD

**Discussion by nosbig** [Kovah/LinkAce#824] [Ideas & Feature Requests]
I would love to be able to have a built-in search for links with zero tags.  This would allow for easy finding tags which need further classification when adding links more than just one or two in a short period.

**Comment by Kovah** [comment]
There is a "without tags" options on the search page. Doesn't this work for you?

**Comment by nosbig** [comment]
It would work if you didn't also need to include a search term.  In the
current implementation, you need to have a non-empty string in the search
box and check the "without tags" checkmark to get results.

**Reply by Kovah** [reply]
This feature is now available in LinkAce 1.15.2

**Reply by caplam** [reply]
When importing links they are automatically tagged. These tags can be useful but prevent from searching untagged links.
It could be useful to filter the list displayed by removing all "import-****" tags
